cmd/xl-zones.go

```diff
@@ -1533,9 +1534,58 @@ func (z *xlZones) GetMetrics(ctx context.Context) (*Metrics, error) {
 	return &Metrics{}, NotImplemented{}
 }
 
+func (z *xlZones) getZoneAndSet(id string) (int, int, error) {
+	for zoneIdx := range z.zones {
+		format := z.zones[zoneIdx].format
+		for setIdx, set := range format.XL.Sets {
+			for _, diskID := range set {
+				if diskID == id {
+					return zoneIdx, setIdx, nil
+				}
+			}
+		}
+	}
+	return 0, 0, errDiskNotFound
+}
+
 // IsReady - Returns true if first zone returns true
```

Comment:
can you fix the comment - it is outdated.

Reply:
This is not true anymore, perhaps we need to elaborate on the logic used internally for future readers.